daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Oh right, my grep failed me. `docker/daemon` imports server, not `daemon/daemon`.
We could setup networkController in `docker/daemon` instead... which is what we do for registry already.